doc/releases/changelog-0.27.0.md

```diff
@@ -174,32 +256,112 @@
 
   Note that this change depends on `jax.pure_callback`, which requires `jax>=0.3.17`.
 
-* The `QNode` class now accepts an `auto` interface, which automatically detects the interface of the given input.
-  [(#3132)](https://github.com/PennyLaneAI/pennylane/pull/3132)
+<h4>The `qml.pauli` module 👀</h4>
 
-  ```python
-  dev = qml.device("default.qubit", wires=2)
-  @qml.qnode(dev, interface="auto")
-  def circuit(weight):
-      qml.RX(weight[0], wires=0)
-      qml.RY(weight[1], wires=1)
-      return qml.expval(qml.PauliZ(0))
+* We've reorganized and grouped everything in PennyLane responsible for manipulating Pauli operators into a `pauli` module. The `grouping` module has been deprecated as a result, and logic was moved from `pennylane/grouping` to `pennylane/pauli/grouping`.
+  [(#3179)](https://github.com/PennyLaneAI/pennylane/pull/3179)
 
-  interface_tensors = [[0, 1], np.array([0, 1]), torch.Tensor([0, 1]), tf.Variable([0, 1], dtype=float), jnp.array([0, 1])]
-  for tensor in interface_tensors:
-      res = circuit(weight=tensor)
-      print(f"Result value: {res:.2f}; Result type: {type(res)}")
+* `qml.pauli.PauliWord` and `qml.pauli.PauliSentence` can be used to represent tensor products and linear combinations of Pauli operators, respectively. These provide a more performant method to compute sums and products of Pauli operators.
+  [(#3195)](https://github.com/PennyLaneAI/pennylane/pull/3195)
+
+  - `qml.pauli.PauliWord` represents tensor products of Pauli operators. We can efficiently multiply and extract the matrix of these operators using this representation.
+
+    ```pycon
+    >>> pw1 = qml.pauli.PauliWord({0:"X", 1:"Z"})
+    >>> pw2 = qml.pauli.PauliWord({0:"Y", 1:"Z"})
+    >>> pw1, pw2
+    (X(0) @ Z(1), Y(0) @ Z(1))
+    >>> pw1 * pw2
+    (Z(0), 1j)
+    >>> pw1.to_mat(wire_order=[0,1])
+    array([[ 0,  0,  1,  0],
+          [ 0,  0,  0, -1],
+          [ 1,  0,  0,  0],
+          [ 0, -1,  0,  0]])
+    ```
+
+  - `qml.pauli.PauliSentence` represents linear combinations of Pauli words. We can efficiently add, multiply and extract the matrix of these operators in this representation.
+
+    ```pycon
+    >>> ps1 = qml.pauli.PauliSentence({pw1: 1.2, pw2: 0.5j})
+    >>> ps2 = qml.pauli.PauliSentence({pw1: -1.2})
+    >>> ps1
+    1.2 * X(0) @ Z(1)
+    + 0.5j * Y(0) @ Z(1)
+    >>> ps1 + ps2
+    0.0 * X(0) @ Z(1)
+    + 0.5j * Y(0) @ Z(1)
+    >>> ps1 * ps2
+    -1.44 * I
+    + (-0.6+0j) * Z(0)
+    >>> (ps1 + ps2).to_mat(wire_order=[0,1])
+    array([[ 0. +0.j,  0. +0.j,  0.5+0.j,  0. +0.j],
+          [ 0. +0.j,  0. +0.j,  0. +0.j, -0.5+0.j],
+          [-0.5+0.j,  0. +0.j,  0. +0.j,  0. +0.j],
+          [ 0. +0.j,  0.5+0.j,  0. +0.j,  0. +0.j]])
+    ```
+
+<h4>New basis rotation and tapering features in `qml.qchem` 🤓</h4>
```

Comment:
Do the backticks show up ok on the website?

Reply:
Hmm best take them out I think